### DISCUSSION TITLE
I am unable to create GitHub Code space

### DISCUSSION THREAD

**Discussion by Shashank-GK** [community/community#35981] [Codespaces] [answered]
I am unable to create My GitHub Codes paces in VS codes

**Comment by jostasik** [comment, marked as answer]
#### To see if codespaces are available to you try the folllowing:

 1 - Navigate to the [template repository](https://github.com/github/haikus-for-codespaces) and select Use this template.

 2 - Select an owner for the new repository, enter a repository name, select your preferred privacy setting, and click Create repository from template.

 3 - On the main page of the newly created repository, click the  Code button and select the Codespaces tab.

  ![image](https://user-images.githubusercontent.com/31175268/195532522-ff4224da-ba41-4748-ad07-554cb4812bec.png)

 4 - On the Codespaces tab, click Create codespace on main.

_**If you don’t see this tab, GitHub Codespaces isn't available for you. For more information about access to GitHub Codespaces, see "[Creating a codespace](https://docs.github.com/en/codespaces/developing-in-codespaces/creating-a-codespace#access-to-github-codespaces)."**_

Otherwise, whenever in a repository you can just press the "." key and it should open right up.


### EDIT: i just saw that codespaces have had issues today, this might also be the cause? https://www.githubstatus.com/incidents/7hfyz8mjg663
